src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_params.py

```diff
@@ -44,9 +44,11 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice', 'resource_group_name', arg_type=resource_group_name_type)
 register_cli_argument('appservice', 'location', arg_type=location_type)
 
+register_cli_argument('appservice list-locations', 'linux_workers_enabled', action='store_true', help='support hosting webapp on linux worker')
```

Comment:
This help text isn't clear to me. Does this mean if you supply this flag, you only see locations that support Linux webapps?